ChatCommander.lua

```diff
@@ -444,6 +456,8 @@ local function afk_casino_tick()
         force_roulette_area()
         force_rig_roulette()
         force_rig_roulette()
+        force_rig_blackjack()
+        force_rig_blackjack()

```

Comment:
Also would like to know why calling the function twice per tick is done here @hexarobi ?

Reply:
I dont remember =) maybe its not needed